src/ui/app.js

```diff
@@ -466,7 +466,7 @@ class ArchiveWebApp extends ReplayWebApp
 
   renderImportModal() {
     return html`
-    <wr-modal style="--modal-width: 740px" @modal-closed="${() => this.showImport = false}" title="Import an Existing Archive">
+    <wr-modal style="--modal-width: 740px" @modal-closed="${() => this.showImport = false}" title="Import Files">
```

Comment:
```suggestion
    <wr-modal style="--modal-width: 740px" @modal-closed="${() => this.showImport = false}" title="Import File">
```
I think we only allow one at a time and elsewhere we seem to use singular File